packages/fxa-content-server/server/templates/pages/src/404.html

```diff
@@ -23,10 +23,10 @@
           <h1 id="fxa-404-header">{{#t}}Page not found{{/t}}</h1>
         </header>
 
-        <section>
+        <p style="line-height: 1.5em;">
```

Comment:
Hey @SrilalS, thanks for the PR! Sorry for the delay in getting back to you, I noticed something funky with the 404 file you edited in the `fxa-payments-server` and wanted to dig around a bit, filed an issue for it here if you're curious: #6168 

We don't typically use inline styles so I think we should use a class here instead, especially to reuse this in the `fxa-payments-server`. It's not super apparent where this should live, but let's add it in the same place we're adding styles for `#fxa-404-home`, inside of `_layout.scss`. Our naming conventions here are also not very strict... We could follow the same convention as that ID, such as:

```
.fxa-404-p {
  line-height: 1.5em;
  margin: 0;
}
```

I noticed the `p` tag adds a larger top/margin bottom than we had with the `section` tag, so I've addressed that here. You can set the class on the `p` tag in both files. 🙂